sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/implementation/VersionTest.java

```diff
@@ -0,0 +1,25 @@
+package com.azure.monitor.opentelemetry.exporter.implementation;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class VersionTest {
+    @Test
+    void testArtifactName() {
+        assertThat(Version.getArtifactName()).isEqualTo("azure-monitor-opentelemetry-exporter");
```

Comment:
Use Junit's assertions instead of assertj library APIs.
`Assertions.assertEquals("azure-monitor-opentelemetry-exporter", Version.getArtifactName());`